tests/basic_one_way_send.go

```diff
@@ -43,39 +54,24 @@ func BasicOneWaySend() {
 		AllowedRelayerAddresses: []common.Address{},
 		Message:                 []byte{1, 2, 3, 4},
 	}
-	signedTx := utils.CreateSendCrossChainMessageTransaction(ctx, subnetAInfo, sendCrossChainMessageInput, fundedAddress, fundedKey, teleporterContractAddress)
 
-	log.Info("Sending Teleporter transaction on source chain", "destinationChainID", subnetBInfo.BlockchainID, "txHash", signedTx.Hash())
-	receipt := utils.SendTransactionAndWaitForAcceptance(ctx, subnetAInfo.ChainWSClient, signedTx)
-
-	bind, err := teleportermessenger.NewTeleporterMessenger(teleporterContractAddress, subnetAInfo.ChainWSClient)
-	Expect(err).Should(BeNil())
-	event, err := utils.GetEventFromLogs(receipt.Logs, bind.ParseSendCrossChainMessage)
-	Expect(err).Should(BeNil())
-	Expect(event.DestinationChainID[:]).Should(Equal(subnetBInfo.BlockchainID[:]))
-
-	teleporterMessageID = event.Message.MessageID
+	var receipt *types.Receipt
+	receipt, teleporterMessageID = utils.SendCrossChainMessageAndWaitForAcceptance(
```

Comment:
no need extra var declaration for both these variables, we can delcare new ones here with `:=`